dspace-server-webapp/pom.xml

```diff
@@ -499,6 +499,12 @@
         <dependency>
             <groupId>org.dspace</groupId>
             <artifactId>dspace-sword</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>xml-apis</groupId>
+                    <artifactId>xml-apis</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
See comment at the other pom.xml

Reply:
See above comment...this change seems to be related to an issue with using Eclipse + DSpace.  It is unrelated to the S3 changes but I expect @abollini made the change to allow his Eclipse IDE to work better in developing this fix.